src/main/java/com/example/app/entity/Role.java

```diff
@@ -0,0 +1,28 @@
+package com.example.app.entity;
+
+import java.util.Collection;
+
+import javax.persistence.*;
+
+import lombok.Data;
+
+/**
+ * Role entity.
+ */
+
+@Entity
+@Table(name = "role")
+@Data
```

Comment:
Consider swapping @Data and @Entity for better readability.

Reply:
Agree, fixed.